packages/visualizations/src/components/MapPoi/Map.ts

```diff
@@ -135,13 +141,36 @@ export default class MapPOI {
 
     private bindedResetLeftPaddingPopup = this.resetLeftPaddingPopup.bind(this);
 
+    /** Update popup display between tooltip, sidebar and modal mode */
+    private updatePopupDisplay(display: PopupDisplayTypes) {
+        // Navigation controls could have been removed if the previous popup display was 'modal'.
+        this.queue((map) => map.addControl(this.navigationControl));
+
+        // Remove all popup display classname modifier before adding the current modifier.
+        Object.keys(POPUP_DISPLAY_CLASSNAME_MODIFIER).forEach((d) => {
+            this.popup.removeClassName(POPUP_DISPLAY_CLASSNAME_MODIFIER[d as PopupDisplayTypes]);
+        });
+        this.popup.addClassName(POPUP_DISPLAY_CLASSNAME_MODIFIER[display]);
+
+        /*
+         * Remove navigation controls when popup display is 'modal'.
+         * When the popup closes, add navigation controls to the maps.
+         */
+        if (display === POPUP_DISPLAY.modal) {
+            this.queue((map) => map.removeControl(this.navigationControl));
+            this.popup.once('close', () =>
+                this.queue((map) => map.addControl(this.navigationControl))
+            );
+        }
```

Comment:
I have a crash in control, I haven't found yet if this is related to this part or somewhere else, do you have it also ? 

![Capture d’écran 2023-11-28 à 10 54 08](https://github.com/opendatasoft/ods-dataviz-sdk/assets/84501475/6ef96d5b-8488-4929-89e3-247ff0c26dac)

Reply:
Should not happens since in https://github.com/opendatasoft/ods-dataviz-sdk/pull/205/commits/7635413b7ebebc37d6388109f1b126cb5b1426d7